container/sagemaker/python_service.py

```diff
@@ -157,6 +159,24 @@ def on_post(self, req, res):
                 'error': str(e)
             }).encode('utf-8')
 
+    def _read_model_config(self):
+        models = []
+        name_key = '    name: '
```

Comment:
The whitespace feels brittle to me. Is there a more maintainable way to do this? Perhaps a regex search?

Reply:
I replaced this with regex search here as well as below.